fe/src/main/cup/sql_parser.cup

```diff
@@ -338,6 +338,7 @@ nonterminal ArrayList<String> opt_plan_hints;
 nonterminal ArrayList<String> opt_sort_hints;
 nonterminal Expr sign_chain_expr;
 nonterminal Qualifier union_op;
+nonterminal String opt_agg_open_hints;
```

Comment:
change this name? This can be hint for other purpose.
I think it should be a List<Sting>, cause we can support other hints later.

Reply:
Ok